Add CHANGELOG for v0.3.2 (#26)
turbot/steampipe-plugin-csv: CHANGELOG.md

## v0.3.2 [2022-06-22]

_Bug fixes_

> - Fixed the issue in the csv file tables where the column names in the header row containing a period returned `null`. ([#25](https://github.com/turbot/steampipe-plugin-csv/pull/25))

Reviewer: ```suggestion
- Fixed an issue in the `{csv_filename}` tables where values would always be `null` for header columns with a period in their name. ([#25](https://github.com/turbot/steampipe-plugin-csv/pull/25))
```
